shared/elements/gaia_sim_picker/script.js

```diff
@@ -10,6 +10,13 @@
  * Dependencies: gaia-menu web component.
  */
 window.GaiaSimPicker = (function(win) {
+  var l10n = navigator.mozL10n || document.l10n;
```

Comment:
I'll check it out.  In https://bugzilla.mozilla.org/show_bug.cgi?id=1204660 we introduced a temporary `mozL10n` proxy which should make working with shared code easier.  Hopefully it'll help here as well.